wd_helper.au3

```diff
@@ -1395,14 +1398,15 @@ Func _WD_UpdateDriver($sBrowser, $sInstallDir = Default, $bFlag64 = Default, $bF
 						__WD_CloseDriver($sDriverEXE)
 						FileDelete($sInstallDir & $sDriverEXE)
```

Comment:
I think it would be best if these lines were moved to just before the `For $FileItem In $FilesInZip` line. This should prevent the scenario where the driver is deleted and not replaced.

Reply:
Take a look here:
[FileDelete($sInstallDir & $sDriverEXE) moved](https://github.com/Danp2/au3WebDriver/pull/204/commits/14882c7ee4dea5cccb60692d929de6fec55baaf5)

Reply:
Take a look here: [comments about FileDelete](https://github.com/Danp2/au3WebDriver/pull/204/commits/3f810fbab37ebd14a66f76e2c071366a7356db1a)